specification/automation/resource-manager/Microsoft.Automation/preview/2017-05-15-preview/job.json

```diff
@@ -0,0 +1,613 @@
+        "x-ms-examples": {
+          "Get Job Output": {
+            "$ref": "./examples/job/getJobOutput.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/automationAccountName"
+          },
+          {
+            "name": "jobName",
+             "description": "The name of the job to be created.",
+             "type": "string",
+             "required": true,
+             "in": "path"
+          },          
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/clientRequestId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2015-10-31/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/jobs/{jobName}/runbookContent": {
+      "get": {
+        "tags": [
+          "Job"
+        ],
+        "operationId": "Job_GetRunbookContent",
+        "description": "Retrieve the runbook content of the job identified by job name.",
+        "produces": [
+          "text/powershell"
+        ],
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/joboperations"
+        },
+        "x-ms-examples": {
+          "Get Job Runbook Content": {
+            "$ref": "./examples/job/getJobRunbookContent.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/automationAccountName"
+          },
+          {
+            "name": "jobName",
+            "description": "The job name.",
+            "type": "string",
+            "required": true,
+            "in": "path"
+          },          
+          {
+            "$ref": "../../stable/2015-10-31/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/clientRequestId"
+          }
+        ],
+        "responses": {
```

Comment:
schema?

Reply:
This will be a string which represents the script used to run the job. As above, there is no schema as it is a script.